CIP-066/README.md

```diff
@@ -0,0 +1,426 @@
+          "3j4yjLxPUjRVtBoJoekB1NJG4o84TzcVohpi5naLABtCp1K8tiNXvRdCafXFcZBh",
+          "VmFJS1mp4vnyDqHsLzrbbQWsdD8SSNoXoArMrihZj4xwneUC2mE2BFeGic58kx84"
+        ]
+      ],
+      "publicKey": [
+        [
+          "MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAowyU5YxcEYGUdJO/7XZw",
+          "aymotFny7qocYQPRnFDUDXwLwLoMFpiTeUKBX79dULeTPMHma2m3JByzqeSMfEaL",
+          "bxeIuIctyG9ZiCnSEx2mGEpKIsHwgGcqPNfdfa0gwXZerWtH5+6YgSuLW49L5Byx",
+          "Tn9fF77gnOqrj4xRriCAneQrZdagi+rk6QoLPEYbM7Ctvw4x59scYf3Lo0Ak495T",
+          "IbjjyFo6mq+bMJWLAmSsaeSX5H3kuZTOcv5h4VloRlPiScuuVLotP3cH2afvM8wp",
+          "fsRe1TOgZLO7azCv6xb7leIUmBYDGDgctHvxmN/bUvw1iflOtHkMH/EeXwPQ7BBQ",
+          "3+NHPLhWvygrOXeBNhJKq1uZCyJQRKcVP6z0r/McEvjRiJOAffvD0YN10n+C4eAP",
+          "byW9h+fu3e2D4gnjLVlXFCdVayEJt0nK0PTMzqfTeYmcDfJn+yj9nQzHytq4WqIr",
+          "vo7wn/goKUMiqQwy9YVgyu89EgswaDFqq9r/LTrjca7O84Jf7s2kCv2qpsuuCF+4",
+          "IqI1r3b0HxisKVNKKcl81rFgL9Uf9pkqUg4k7Tty+iiI7kfv3yld9ESLO5bSlHcU",
+          "rJzBm2agIyPUdnXnUg+CbBONFgY7ayWYnmAcCBBs/dTFrcUx7guhl4IPXY/8MMjS",
+          "rZqb/5Uml4KdVHHUDaBnkQ8CAwEAAQ=="
+        ]
+      ]
+    }
+  ]
+}
+```
+
+#### 2. Collection Token
+
+```
+{
+  "725":{
+    "version": "1.0",
+    "fdb275fb75db33313626e0361fc763b6995e97db312be87590e48aa1": {
+        "CIP0066Test": {
+            "type": "Ed25519VerificationKey2020",
+            "files": [
+                {
+                    "src": "ipfs://QmacjTQe5q1ur2FbqW9TehScoP7cZEcAWnSA6gz4FEEMi2",
+                    "name": "CIP-0066_NMKR_IAMX",
+                    "mediaType": "application/ld+json"
+                }
+            ],
+            "@context": "https://github.com/IAMXID/did-method-iamx"
+        }
+    }
+  } 
+}
+```
+
+## Cross-Chain Compatibility
+
+To make this standard cross-chain compatible it is possible to set the value of the src parameter under files to a unique identifier on another blockchain, where the DID might be located.
+
+## Backwards Compatibility
+
+To keep NFT metadata compatible with changes coming up in the future, we use the **`version`** property.
+
+## Rationale
+The focus of this standard is on ease-of-use & flexibility. Using this standard it is possible to reference DIDs not only on Cardano but at any arbitrary location, even cross-chain. It is also relatively simple to understand and implement because it borrows heavily from pre-existing metadata standards.
+
+## Path to Active
+
+[IAMX](https://iamx.id/) and [NMKR](https://www.nmkr.io/) have created a reference implementation of the standard that can be viewed [here](https://cardanoscan.io/transaction/608dd4fba7df5c523a1dfdca8d48fd1d69e7e1995a342ceb01c8355c62a0845b?tab=metadata).
+
+[IAMX](https://iamx.id/) will be creating a tool to create the DIDs that will be integrated natively into [NMKR](https://www.nmkr.io/) studio.
+
+By displaying this solution at the Catalyst Guild Hall we hope to reach more creators that are willing to integrate this solution.
```

Comment:
If a reference implementation exist, I'd strongly suggest to include some test vectors as examples in this section to help other implementations to also happen. As mentioned also in some other comments, the data is ultimately stored on-chain as binary (CBOR). Thus, test vectors should include **at least** a CBOR representation of the metadata, and possibly a JSON / YAML one for human-readability.